gpt_engineer/ai.py

```diff
@@ -271,6 +271,27 @@ def format_token_usage_log(self) -> str:
             result += str(log.total_tokens) + "\n"
         return result
 
+    def usage_cost(self) -> float:
+        """
+        Return the total cost in USD of the api usage.
+
+        Returns
+        -------
+        float
+            Cost in USD.
+        """
+        prompt_price = 0.03
+        completion_price = 0.06
```

Comment:
Prices should not be hardcoded what if they change prices in future ?

Reply:
I agree that it is not ideal, but getting it with an api call would be uneccesary overhead.
[Langchain also hardcodes](https://github.com/langchain-ai/langchain/blob/116cc7998cac563894bdf0037259db767882b55c/libs/langchain/langchain/callbacks/openai_info.py#L7C1-L7C25)

But maybe it is better to get if from langchain as they might be more reliable in updating it.

Reply:
Updated to get the price from langchain.

Reply:
I still think its better to get prices for each model from the API or use some cache mechanism to store the prices and keep on eye on API prices changes.